frontend/my-app/src/AdminPanelButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {useNavigate} from "react-router-dom";
+import {apiUrl} from "./api";
+import {FormEvent} from "react";
+
+export const AdminPanelButton = () => {
+
+        const navigate  = useNavigate()
+        const FormSubmit = async (e: FormEvent) => {

```

Comment:
name method with lowercase first letter. Use verbs, not nouns. `handleFormSubmission`